packages/components/src/products.tsx

```diff
@@ -270,12 +270,14 @@ export const PRODUCTS_MENU_LIST: MenuItem['items'] = Object.fromEntries(
         newWindow: true,
         title: (
           <div className="flex items-center gap-2">
-            <Logo
+            <div
               className={cn(
-                'size-4 translate-y-[0.25px]',
-                i > 3 && 'rounded-sm bg-gray-500 text-[8px] text-white dark:bg-white/10',
+                'flex size-4 translate-y-[0.25px] items-center justify-center',
+                i > 6 && 'rounded-sm bg-gray-500 text-white dark:bg-white/10',
               )}
-            />
+            >
+              <Logo className="size-4 text-[8px]" />
```

Comment:
this piece of code is overcomplicated to me

the container defines `flex size-4 items-center justify-center` but children have `size-4` so technically you can remove everything from the container

and `translate-y-[0.25px]` do human eye see this difference of 1/4 pixel?

Reply:
> and translate-y-[0.25px] do human eye see this difference of 1/4 pixel?

it was uneven to me without it. Take a look at the video.

https://github.com/user-attachments/assets/1e2cd05e-67bb-4314-89ae-e9c3182749cb

I have a 2021 MacBook with 14.2-inch (diagonal) Liquid Retina XDR display; 3024x1964 native resolution at 254 pixels per inch, but I'm running it at 1512x982 resolution, so at least every 0.5px should be visible, but empirically I feel it's 0.25px. 

You an go lower, but AFAIK browsers can do some rounding and the effects may differ across engines.

https://www.simonbattersby.com/blog/browsers-and-fractional-pixels/

---

> so technically you can remove everything from the container

Good catch, not everything though.
This is how it looks when you remove everything.

<img width="430" alt="image" src="https://github.com/user-attachments/assets/a7eb4468-8dbd-40eb-8ed2-6d58ba46d4cf">
<img width="373" alt="image" src="https://github.com/user-attachments/assets/10befa09-5d3b-4ce5-af8e-0a973c466483">

It's still fine if we remove everything but `flex`. We need it for the box not to grow to 24px height.